ui/packages/catalog-ui-search/src/main/webapp/dev/component/card-guide/card-guide.view.js

```diff
@@ -39,6 +41,35 @@ module.exports = BaseGuideView.extend({
         }));
         this.workspaceExample.currentView.activateGridDisplay();
     },
+    showResult2Example() {
+        this.result2Example.show(new ResultItemView({
+            model: new QueryResultModel({
+                actions: [{
+                    description: 'example',
+                    id: 'example',
+                    title: 'example',
+                    url: 'https://www.google.com'
+                }],
+                distance: null,
+                hasThumbnail: false,
+                isResourceLocal: true,
+                metacard: {
+                    id: 'blah blah blah',
```

Comment:
✏️ this is a nit, but I think we should clean up this example metacard. 
**id** and **validation-warnings** should be more representative
**resource-download-url** we typically use https://example.com for this sort of thing